src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -6425,8 +6430,9 @@ def failover_group_create(
         server_name,
         failover_group_name,
         partner_server,
+        secondary_type=None,
         partner_resource_group=None,
-        failover_policy=FailoverPolicyType.automatic.value,
+        failover_policy=FailoverPolicyType.manual.value,
```

Comment:
Parameter's default value change is considered as breaking change. We only allow breaking changes twice a year and next breaking change window is Ignite Sprint

Reply:
So, what should be the next steps? Should i revert the change or this PR will be part of next Nov release? I am fine with Nov release.